.github/workflows/pull_request.yml

```diff
@@ -32,22 +32,5 @@ jobs:
           ruby-version: "2.6"
           bundler-cache: true
```

Comment:
removed the caching steps as the gem action documentation suggests that setting bundler-cache: true will cache your gems (provided they are in the default Gemfile location).

Reply:
Since you are touching this, can you pull out the testing into it's own callable workflow file, similar to 
the way lupo uses the same CI testing in pull requests/deployment/release

Ex:
https://github.com/datacite/lupo/blob/master/.github/workflows/pull_request.yml
and
https://github.com/datacite/lupo/blob/master/.github/workflows/parallel_ci.yml
(No need to do the stuff to run the specs in parallel here, they are much smaller/faster)

Then we just have one place to update if we need to update something related to tests/specs.

Reply:
> Not a blocker but a suggestion
> 
> Since you are touching this, can you pull out the testing into it's own callable workflow file, similar to the way lupo uses the same CI testing in pull requests/deployment/release
> 
> Ex: https://github.com/datacite/lupo/blob/master/.github/workflows/pull_request.yml and https://github.com/datacite/lupo/blob/master/.github/workflows/parallel_ci.yml (No need to do the stuff to run the specs in parallel here, they are much smaller/faster)
> 
> Then we just have one place to update if we need to update something related to tests/specs.

thanks @jrhoads. this is a great suggestion. I will get on it.